### PR TITLE
changes spawners, removes unnecessary stuff

### PR DESCRIPTION
**Pull request by jjpark-kb**
## Description

removes the old spawners, put new spawners.
spawners now function differently compared to before.

spawners are no longer destroyabled, but instead cover-able.
using either 4 wooden planks or 4 metal rods will cover it.
you can also uncover the spawners with a crowbar.

rather than being a mob, it is now a structure, which may help with performance minimally.
rather than being ran on a life proc, which is intensive, runs through a subsystem, running every 10 seconds.

removed tons of stuff that used the old spawners:
tendrils, legion the megafauna, etc.
also removed some of the spawners that were not fo13 friendly... such as tg mining mobs.

also had to update all the maps to use the new spawners, which I did.
(I also just updated two other maps, possum and original_pahrump, so that its actually openable)
## Motivation and Context

minimal better performance
better interactions with the spawners
## How Has This Been Tested?

compiled & tested
## Screenshots (if appropriate):
overlays for whatever you used
image
time it takes to remove it
image

## Changelog (necessary)
:cl:
del: Removed old spawners and any connected items to spawners (legion, nether, etc.)
imageadd: added two mob nest overlays
code: changed mob spawners/nests; How they work and what they are
/:cl:

### CODE REVIEW

**Comment by goodolrusty** [comment]
Im reviewing this currently, but I believe it to be okay at a glance, alots moving here and we'll discuss it further tomorrow.

**Comment by jjpark-kb** [comment]
Okay, please, the vision and direction of the server need to be made clear, because Im losing my patience over making stuff that takes time, only for it to not really be considered because of something.

I heard that we wanted to focus on fo13. Why does this mean we would keep ss13 stuff?
I removed legion because it is a mob spawner, which I completely reworked. 
Why do we want goliaths, hivelords, etc. in a fo13 server?

This rework would have had some better performance compared to the other mobspawners.

Im closing this because I dont want to deal with it anymore. Players complain and I get flak.

**Comment by jjpark-kb** [comment]
Okay here is my reasoning:

I don't think its smart to keep code that isn't used. Code bloat is a very bad, very bad thing when it comes to codebases.
An example of extreme code bloat is /vg/. If I try to load that codebase up, it takes me forever to open compared to other codebases. It isn't just me, but everyone who would try to open it up would notice how long it takes compared to other codebases.

We don't want to deter people from contributing because its too much of a chore opening up a single file.

Now, I removed tendrils, a portal, and other stuff because it isnt at all fallout related. We will probably never use those because it breaks immersion, isn't lore friendly, etc. So there should be no issue when I remove those.

The next thing is the legion removal. It isnt the megafauna removal, its the hivelord/legion, the small one, that is being removed. This goes back to code bloat. I removed the base mob spawner because its useless to keep it when I basically revamped it to be better for performance with almost no change in function (besides the changes I did do to function). It would be literally impossible to ever have legion/hivelord with this PR, which I believe could be really good for the server.

So again, if you really want all that non-fallout stuff, tell me and I'll close this PR again. Otherwise, I'm putting it back up.

**Comment by jjpark-kb** [comment]
there, I added back your stuff, just uncommented. 

Im warning you right now, code bloat will hurt development. Not my call on whether something is code bloat, just warning you...

File: code/game/objects/structures/mob_spawners.dm [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+	mob_types = list(/mob/living/simple_animal/hostile/deathclaw = 5, 
+					/mob/living/simple_animal/hostile/deathclaw/mother = 1)
+
+/obj/structure/nest/scorpion
+	name = "scorpion nest"
+	mob_types = list(/mob/living/simple_animal/hostile/radscorpion = 1)
+
+/obj/structure/nest/radroach
+	name = "radroach nest"
+	mob_types = list(/mob/living/simple_animal/hostile/radroach = 1)
```
Comment (Laus4Deus) [inline code/game/objects/structures/mob_spawners.dm:132]: So this is a complete rework of the spawners into `nest` Cool!

File: code/modules/power/lighting.dm [hunk 2]
```diff
@@ -816,6 +816,23 @@
...
+				glowybit.alpha = CLAMP(light_power*250, 30, 200)
+				add_overlay(glowybit)
+		if(LIGHT_EMPTY)
+			icon_state = "[base_state]-empty"
+		if(LIGHT_BURNED)
+			icon_state = "[base_state]-burned"
+		if(LIGHT_BROKEN)
+			icon_state = "[base_state]-broken"
+	return
+
```
Comment (Laus4Deus) [inline code/modules/power/lighting.dm:835]: Hey, you put in the floorlight fix!
tanks, I coudlnt' because my code didn't compile... *sighs*

Comment (jjpark-kb) [reply code/modules/power/lighting.dm:835]: that isnt intended, removing.

**Pull request merged**